packages/union-component/src/components/DonationFundDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { SelectHTMLAttributes, useEffect, useState } from 'react';
+import DonationDropdown from './DonationDropdown';
+import { getDonationFunds, IFund } from '../utils/funds';
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+  setFundId: (fundId: number) => void;
+}
+
+const DonationFundDropdown: React.FC<Props> = ({
+  id,
+  onChange,
+  required,
+  setFundId,
+  value
+}) => {
+  const [options, setOptions] = useState<IFund[]>([]);
+
+  useEffect(() => {
+    const getOptions = async () => {
+      const funds = await getDonationFunds();
+
+      if (funds?.length) {
+        setOptions(funds);
+        setFundId(Number(funds[0].id));
+      }
+    };
+
+    getOptions();
+  }, [setFundId]);
+
+  return (
+    <>
+      <label htmlFor={id} className="block mb-2 text-gray-100">
+        Would you like to donate to a specific fund?
+      </label>
+      <DonationDropdown
+        id={id}
+        onChange={onChange}
+        required={required}
+        value={value}
+      >
+        {options.map((option) => (
```

Comment:
We should at least create a ticket in asana to address later saying:
_"provide feedback while options are being loaded"_

What I have done before is to add a disabled option saying something like _"loading funds..."_

Reply:
Agree, I will also add some feedback for errros in that ticket